.kitchen.yml

```diff
@@ -1,6 +1,12 @@
 ---
 driver:
   name: vagrant
+  # # Docker support
```

Comment:
Don't really see a good reason to. This way it is easier to switch when commenting out in vim :)